src/elm/Ports.elm

```diff
@@ -59,3 +62,24 @@ port javascriptInPort : (Value -> msg) -> Sub msg
 
 
 port storeLanguage : String -> Cmd msg
+
+
+{-| Store recent searches to the `localStorage`.
+-}
+port storeRecentSearches : String -> Cmd msg
+
+
+{-| Ping JS to send back the recent searches from the `localStorage`.
+-}
+port getRecentSearches : () -> Cmd msg
```

Comment:
👀

Reply:
Sorry, your reaction is too expressive for me to catch 😅 Can you please elaborate? :)

Reply:
just a different way of using ports, instead of a single one, I like the approach actually